src/industry_gui.cpp

```diff
@@ -1547,7 +1535,7 @@ class IndustryDirectoryWindow : public Window {
 
 		for (byte j = 0; j < lengthof(i->produced_cargo); j++) {
 			if (i->produced_cargo[j] == CT_INVALID) continue;
-			cargos.push_back({ i->produced_cargo[j], i->last_month_production[j], cargo_suffix[j].text, ToPercent8(i->last_month_pct_transported[j]) });
+			cargos.push_back({ i->produced_cargo[j], i->last_month_production[j], cargo_suffix[j].text.c_str(), ToPercent8(i->last_month_pct_transported[j]) });
```

Comment:
Could CargoSuffix::text be std::string or is that creeping too much?

Reply:
Well... `CargoSuffix::text` has become `std::string` in this PR.

However, I did, in the end, not change `CargoInfo::suffix` to `std::string`, as that is just a "pointer" to the data and as such a needless copy. I did try making it `std::string_view`, but then I ran into `SetDParamStr` that doesn't accept `std::string_view`, so I stopped and undid the `std::string_view` change as it became a bit too rabbit-holey.

Reply:
Whoops, yes, that's the one I meant :-)

And yes, that definitely sounds like creeping too much!